user-guide/Advanced_Modules/DOM/DOM_objects.md

```diff
@@ -35,3 +35,33 @@ For example, to create a system that stores time spent working on specific tasks
 - Field descriptor 4: *Name* = "Stop time" and *Type* = "DateTime"
 
 A DOM definition will then need to be created that contains a link to this section definition, so that employees can then register time by creating a new DOM instance linked to that DOM definition. The DOM instance will have a section linked to the section definition that contains a field value for each of the field descriptors.
+
+## ITrackBase Properties
+
+From DataMiner 10.3.2/10.4.0 onwards, all DOM objects (including the `ModuleSettings`) contain four properties that reflect who has created/updated the object, and at what time.
+
+- **LastModified** : The moment when the object was last modified, in UTC DateTime format.
```

Comment:
Could you remove the space before the colons in this list? Usually, there's no space before a colon in English, so unless there's a very specific reason to have one after all, we should avoid this.